bloggies_frontend/src/custom.d.ts

```diff
@@ -44,7 +45,10 @@ interface SearchResults {
 
 export interface User {
   id: number,
-  username: string,
+  email: string,
   join_date: string,
   display_name: string
+  membership_status: string,
```

Comment:
Is there a reason why last_submission_date is not in the User Interface?

Reply:
nope! I'll add it in :)